### DISCUSSION TITLE
Changing config in other files than server.properties?

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [itzg/docker-minecraft-server#1079] [Q&A] [answered]
> When the environment variable REPLACE_ENV_IN_PLACE is set to true (the default), the startup script will go through all files inside the container's /data path and replace variables that match the container's environment variables.

Does this mean I can do `-e autosave=1200` to change the `autosave:` field in bukkit.yml?

**Comment by itzg** [comment]
Yes but your example doesn't quite follow the suggested structure. In your case, you'll need to set an empty prefix with `-e REPLACE_ENV_VARIABLE_PREFIX=` and in bukkit.yml you'll need

```
  autosave: ${autosave}
```

**Reply by alvarlagerlof** [reply]
Interestinng, okay. Can I specify only wanting that part of the config file initially and let the rest generate?

**Reply by itzg** [reply]
It (the spigot/bukkit server startup itself) will probably populate the rest of the file, but that's a total guess. Give it a try and see what happens.

**Reply by alvarlagerlof** [reply]
I realized that too. Works out great, thank you!

**Comment by alvarlagerlof** [comment, marked as answer]
I ended up providing config files with only the properties I wanted to change and let the rest be filled by Bukkit/Spigot/Paper
